packages/1password/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.2.0"
+  changes:
+    - description: Add new "event.action" to item usage events
```

```suggestion
    - description: Add new "event.action" to item_usages events.
```

That looks like the name being used by the data stream.